config/database.travis.yml

```diff
@@ -0,0 +1,7 @@
+test: &test
+  adapter: postgresql
+  encoding: unicode
+  database: wbooks_test
+  pool: 5
+  username: rails-bootstrap
```

Comment:
I would change this username and password by wbooks (here and in the .travis.yml file)